app/models/backend_api.rb

```diff
@@ -29,6 +29,8 @@ class BackendApi < ApplicationRecord
 
   has_system_name(uniqueness_scope: [:account_id])
 
+  scope :orphans, -> { joining { services.outer }.where('backend_api_id IS NULL') }
```

Comment:
```suggestion
  scope :orphans, -> { joining { services.outer }.where{ BabySqueel[:backend_api_configs].backend_api_id == nil } }
```
I do not know why this does not work

```ruby
joining { services.outer }.where{ backend_api_configs.backend_api_id == nil }
```

It constructs 🤷‍♂ 

```sql
SELECT `backend_apis`.* FROM `backend_apis` LEFT OUTER JOIN `backend_api_configs` ON `backend_api_configs`.`backend_api_id` = `backend_apis`.`id` LEFT OUTER JOIN `services` ON `services`.`id` = `backend_api_configs`.`service_id` WHERE `backend_api_configs_backend_apis`.`backend_api_id` IS NULL
```

See the `WHERE` clause ➡️  `WHERE `backend_api_configs_backend_apis`.`backend_api_id` IS NULL`

Reply:
Because it's a `LEFT JOIN` and the `BackendApi` no longer exists.

Reply:
it worked.. I tested here.